src/WorkPage/WorkPage.css

```diff
@@ -8,32 +8,46 @@
 }
 
 .WorkPage-header {
-  display: flex;
-  flex-wrap: wrap;
-  gap: 0.5em;
-  margin: 0;
-  font-size: 3em;
-  font-weight: 550;
-  align-items: baseline;
-  justify-content: center;
+	display: flex;
+	flex-wrap: wrap;
+	gap: 0.5em;
+	margin: 0;
+	font-size: 3em;
+	font-weight: 550;
+	align-items: baseline;
+	justify-content: center;
+	background-image: url('../img/bg2.png');
+	background-size: cover;
+	background-position: center;
+	color: #E0E0E0;
+	padding: 1em;
+	border: 2px solid #ffffff;
+	border-radius: 30px;
+	box-shadow: 0 4px 8px rgba(0, 0, 0, 0.5);
+	overflow: hidden;
 }
 
 .WorkPage-go-button {
-  border: solid 4px #1b1e24;
-  border-radius: 15px;
-  height: 65px;
-  width: 70px;
-  display: flex;
-  align-items: center;
-  justify-content: center;
-  font-size: 2em;
+	border: solid 2px white;
+	border-radius: 15px;
+	height: 65px;
+	width: 70px;
+	display: flex;
+	align-items: center;
+	justify-content: center;
+	font-size: 2em;
+	background-image: url('../img/bg2.png');
+	background-size: cover;
+	background-position: center;
 }
 
-.WorkPage-go-button:hover {
-  background-color: #1b1e24;
-}
+	.WorkPage-go-button:hover {
+		background-color: #1b1e24;
+		opacity: 0.7;
+	}
 
-.WorkPage-go-button:disabled {
-  color: grey;
-  background-color: #1b1e24;
-}
+	.WorkPage-go-button:disabled {
+		color: grey;
+		background-color: #1b1e24;
```

Comment:
since there's a background-image now, you can probably remove the `background-color` attribute.